package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "babel-brunch",
-  "version": "5.1.2",
+  "version": "5.2.0",
```

Comment:
Please don't bump package.json version in a PR, that happens during a release.

It will be going to 6.0 anyway to align with babel.